src/Form/DataTransformer/ModelToIdPropertyTransformer.php

```diff
@@ -104,6 +117,9 @@ public function reverseTransform($value)
         return $collection;
     }
 
+    /**
+     * @return string|mixed
+     */
     public function transform($entityOrCollection)
```

Comment:
This param should be T[]|T|null?

Reply:
Not sure if it has to be a collection, the code seems to say it could be only an array